ui/src/components/Treasury.jsx

```diff
@@ -88,45 +87,40 @@ function VaultList() {
     );
   }
 
-  const makeButtons = (key, vaultData) => {
-    // TODO: use a less fragile way of keeping track of a loan status
-    const canManage = vaultData.status === 'Loan Initiated';
-    return (
-      <CardActions>
-        <Grid container justify="flex-end">
-          <Grid item className={classes.button}>
-            <Button
-              variant="contained"
-              color="primary"
-              onClick={() => handleOnClick(key)}
-              disabled={!canManage}
-            >
-              Manage Vault
-            </Button>
-          </Grid>
-        </Grid>
-      </CardActions>
-    );
-  };
-
   return (
     <div>
       <ErrorBoundary>
         <Grid container className={classes.grid} alignItems="stretch">
-          {vaultsList.map(([key, v]) => (
-            <Grid item key={key} className={classes.gridCard}>
-              <Card key={key} className={classes.card}>
-                <CardContent>
-                  <VaultSummary
-                    vault={v}
-                    id={key}
-                    brandToInfo={brandToInfo}
-                  ></VaultSummary>
-                </CardContent>
-                {makeButtons(key, v)}
-              </Card>
-            </Grid>
-          ))}
+          {vaultsList.map(([key, v]) => {
```

Comment:
Seemed to me like the function up top was superfluous when you could just add it in here and set the variable above the return within the JSX